torch/fx/experimental/symbolic_shapes.py

```diff
@@ -2283,14 +2283,25 @@ def create_unbacked_symbool(self):
 
         return SymBool(SymNode(sympy.Eq(symbol, 1), self, bool, None, fx_node=fx_node))
 
+    def create_unspecified_symbol(
+        self,
+        val: int,
+        source: Source,
+        dynamic_dim: DimDynamic = DimDynamic.DUCK,
+        constraint_dim: DimConstraint = None,  # NB: includes None
+    ) -> "sympy.Expr":
+        return self.create_symbol(val, source, dynamic_dim, constraint_dim, positive=None)
+
     def create_symbol(
         self,
         val: int,
         source: Source,
         dynamic_dim: DimDynamic = DimDynamic.DUCK,
         constraint_dim: DimConstraint = None,  # NB: includes None
+        positive: Optional[bool] = True,
```

Comment:
Is this really Optional[bool]? What does None mean?

Reply:
It seems you can set this to False to mean "never positive", but we don't actually have any concrete use case for this, AFAIK

Reply:
That's true. I did this so that I wouldn't need to `positive = positive if positive else None` when I'm actually creating the symbol. Maybe I could just leave a comment when I call it with `positive=None`. What do you think?

Reply:
I think a comment is OK too, just make it hard for people to accidentally say `positive=False` when they meant `positive=None`

Reply:
Hmmm. Maybe we could just throw in an assert there like:

```python
def create_symbol(
    ...
    positive: Optional[bool] = True
):
    assert positive != False, (
        "Unspecified integers range over negative and positive numbers. "
        "Setting positive=False means strictly negative. "
        "Use positive=None instead."
    )
```

Let me know what you think.